docs/source/recipes/color-picker-widget.md

```diff
@@ -0,0 +1,150 @@
+---
+myst:
+  html_meta:
+    "description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:title": "How to use the color picker widget"
+    "keywords": "Volto, Plone, frontend, React, blocks, forms, widget, color, picker"
+---
+
+# Color picker widget
+
+Volto comes with a color picker widget that can be used in any Volto form.
```

Comment:
Insert a sentence with a link to the Storybook color picker demo.

```suggestion
Volto comes with a color picker widget that can be used in any Volto form.
You can [try a demo of the default color picker](https://6.docs.plone.org/storybook/?path=/story/edit-widgets-colorpicker--default).
```

Reply:
Ah sorry I forgot.